packages/libs/error-stack/Cargo.toml

```diff
@@ -14,32 +14,39 @@ categories = ["rust-patterns", "no-std"]
 
 [dependencies]
 tracing-error = { version = "0.2", optional = true, default_features = false }
-once_cell = { version = "1.10.0", optional = true, default_features = false }
 pin-project = { version = "1.0.10", optional = true, default_features = false }
 futures-core = { version = "0.3", optional = true, default-features = false }
 smallvec = { version = "1", optional = true, default_features = false, features = ['union'] }
 anyhow = { version = "1", default-features = false, optional = true }
 eyre = { version = "0.6", default-features = false, optional = true }
 
+owo-colors = { version = "3.4.0", default-features = false, optional = true, features = ['supports-colors'] }
+
 [dev-dependencies]
 serde = { version = "1.0.137", features = ["derive"] }
 serde_json = "1.0.81"
 futures = { version = "0.3.21", default-features = false, features = ["executor"] }
 trybuild = "1.0.63"
 tracing = "0.1.35"
 tracing-subscriber = "0.3.14"
+insta = { version = "1.18.0", features = ['filters'] }
+regex = "1.6.0"
+expect-test = "1.4.0"
+ansi-to-html = "0.1.0"
 once_cell = "1.13.0"
 
 [build-dependencies]
 rustc_version = "0.4"
 
 [features]
 default = ["std", "small"]
-std = ["anyhow?/std", "once_cell?/std"]
-hooks = ["dep:once_cell", "std"]
+std = ["anyhow?/std"]
 spantrace = ["dep:tracing-error"]
 futures = ["dep:pin-project", "dep:futures-core"]
 small = ["dep:smallvec"]
+pretty-print = ["dep:owo-colors"]
+unstable = []
+hooks = ["std"]
```

Comment:
Bonus: Sort by name 😆

Reply:
fixed in e9e2509b4ef001fae84c1a063ef984098a75fc4b